ebu_tt_live/bindings/_ebuttdt.py

```diff
@@ -464,7 +464,7 @@ class PercentageOriginType(TwoDimSizingMixin, ebuttdt_raw.percentageOriginType):
 
 class PixelExtentType(TwoDimSizingMixin, SizingValidationMixin, ebuttdt_raw.pixelExtentType):
 
-    _groups_regex = re.compile('(?:[+-]?(?P<first>\d*\.?\d+)(?:px))\s(?:[+-]?(?P<second>\d*\.?\d+)(?:px))')
+    _groups_regex = re.compile('(?:[+]?(?P<first>\d*\.?\d+)(?:px))\s(?:[+]?(?P<second>\d*\.?\d+)(?:px))')
```

Comment:
If everyone decides "unsigned" (which would be more restrictive than TTML1) then we would have to remove the `[+]` as well.